frontend/institution/institutionController.js

```diff
@@ -66,6 +66,11 @@
             institutionCtrl.portfolioUrl = url;
         }
 
+        institutionCtrl.limitString = function limitString(string, limit){
```

Comment:
This function already exists, if it's applied to a specific scope extract to Utils.